platform/mellanox/mlnx-platform-api/sonic_platform_api/chassis.py

```diff
@@ -13,7 +13,12 @@
 try:
     from sonic_platform_base.chassis_base import ChassisBase
     from sonic_platform_api.psu import Psu
```

Comment:
Regarding "sonic_platform_api": From my original design (see https://github.com/Azure/SONiC/pull/285/files#diff-64ce147a07eff38b7e6f23dc3d368082R50), the platform-specific implementation of the new platform API package will simply be called "sonic_platform"

Reply:
@jleveque please check the latest commit.